clean-sensitive-data/main.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+public function removeEmailStrings(string inputString) returns string {
+    string:RegExp emailPattern =  re `[A-Za-z0-9\._%+-]+@[A-Za-z0-9\.-]+\.[A-Za-z]{2,}`;
+    return emailPattern.replaceAll(inputString, "");
+}
```

Comment:
Wouldn't it be better to replace the removed pattern with some kind of placeholder instead of an empty string?